meetings/meetings/urls.py

```diff
@@ -13,7 +13,8 @@
 urlpatterns = [
     url(r'^', include(router.urls)),
     url(r'^conferences/', include('conferences.urls', namespace='conferences')),
-    url(r'^checklogin/', apiViews.checkLoggedIn.as_view(), name='checklogin'),
+    url(r'^checklogin/', apiViews.checkLoggedIn.as_view(), name='checklogin'), 
+    url(r'^current/', apiViews.viewCurrentUser.as_view(), name='current'),
```

Comment:
I was thinking the same, I'll do this and then edit the mixin so that it hits this endpoint.